pyro/distributions/testing/rejection_gamma.py

```diff
@@ -26,7 +26,8 @@ def __init__(self, concentration):
         # Compute log scale using Gamma.log_prob().
         x = self._d.detach()  # just an arbitrary x.
         log_scale = self.propose_log_prob(x) + self.log_prob_accept(x) - self.log_prob(x)
-        super().__init__(self.propose, self.log_prob_accept, log_scale)
+        super().__init__(self.propose, self.log_prob_accept, log_scale,
```

Comment:
Some of the distribution tests are failing. I think we may need to change the `.expand` below as well?